main.py

```diff
@@ -1,10 +1,39 @@
 import init_django_orm  # noqa: F401
 
+import json
+
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as file:
+        players = json.load(file)
+    for name, info in players.items():
+        if not Race.objects.filter(name=info["race"]["name"]).exists():
+            Race.objects.create(
+                name=info["race"]["name"],
+                description=info["race"]["description"])
+        player_race = Race.objects.get(name=info["race"]["name"])
+        if (info["guild"] is not None and not
+                Guild.objects.filter(name=info["guild"]["name"]).exists()):
```

Comment:
fix indentation here

Reply:
with any other indentation except this one PyCharm show me PEP8 warning. what exactly indentation do you recommend  ?